README.md

```diff
@@ -230,4 +230,45 @@ simulated exposure. Example use shown in the WFSS notebook.
 
```

Comment:
And while you're at it, I just noticed a typo in the definition of raw MIRAGE output files - "calirbations".

Reply:
Fixed. Thanks.